### PR TITLE
Feature - Add Appointment Modification

### LINKED ISSUE #114: Feature - Add command for Appointment Modification
Add the functionality of modifying appointments currently in ContaX.

### PR DESCRIPTION
**Pull request by sharpstorm**
## Changes
This PR adds a command for modifying appointments previously created.

## Related Issues
- Closes #114

### CODE REVIEW

File: src/main/java/seedu/contax/logic/commands/EditAppointmentCommand.java [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+    public static final String MESSAGE_EDIT_APPOINTMENT_SUCCESS = "Edited Appointment: %1$s";
+    public static final String MESSAGE_NOT_EDITED = "No fields supplied, nothing was changed.";
+    public static final String MESSAGE_OVERLAPPING_APPOINTMENT = "The new appointment will overlap with"
+            + " another appointment in the schedule!";
+
+    private final Index index;
+    private final EditAppointmentDescriptor editAppointmentDescriptor;
+
+    /**
```
Comment (dandaandaaaaaan) [inline src/main/java/seedu/contax/logic/commands/EditAppointmentCommand.java:58]: missed a javadoc header here

**Review by dandaandaaaaaan — APPROVED** [review]
minor nit on javadoc otherwise lgtm!

File: src/main/java/seedu/contax/logic/commands/EditAppointmentCommand.java (hunk 1 above)
Comment (sharpstorm) [reply src/main/java/seedu/contax/logic/commands/EditAppointmentCommand.java:58]: Thanks for spotting it, fixed in commit 3d9af4e.

File: src/test/java/seedu/contax/testutil/DateInputUtil.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package seedu.contax.testutil;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+import seedu.contax.commons.util.DateUtil;
+
+public class DateInputUtil {
```
Comment (KwanHW) [inline src/test/java/seedu/contax/testutil/DateInputUtil.java:9]: Missing javadoc

**Review by KwanHW — APPROVED** [review]
LGTM. Just a minor nit to fix

File: src/test/java/seedu/contax/testutil/DateInputUtil.java (hunk 2 above)
Comment (sharpstorm) [reply src/test/java/seedu/contax/testutil/DateInputUtil.java:12]: Thanks for catching it, fixed in commit f9b8464,

**Pull request merged**